### DISCUSSION TITLE
Making installable version for Windows with sending to imgur disabled or deleted

### DISCUSSION THREAD

**Discussion by Szymaniak** [flameshot-org/flameshot#3412] [Q&A]
Hi 

Is it possible to make installable version for Windows with sending picture to imgur option disabled? Or at least, can I modify portable version and delete sending to imgur option?

**Comment by mmahmoudian** [comment]
There is already a thread about this at #1529, but I think for you tge easiest approach is explained here:

https://github.com/flameshot-org/flameshot/issues/1529#issuecomment-812110134
